HousingFinanceInterimApi.Tests/V1/UseCase/MoveHousingBenefitFileUseCaseTests.cs

```diff
@@ -433,18 +438,18 @@ public async Task UCCopiesValidAcademyFilesWithTheCorrectNewCalculatedName()
             var academyFolders = RandomGen.CreateMany<GoogleFileSettingDomain>(quantity: 1);
             var academyFolderFiles = RandomGen.GoogleDriveFiles(filesValidity: true, count: 2).ToList();
 
-            var oldNameFile1 = "06052022_Something_Academy_20052022";
-            var oldNameFile2 = "11022023_Something_Academy_25022023";
 
-            academyFolderFiles[0].Name = oldNameFile1;
-            academyFolderFiles[1].Name = oldNameFile2;
+            academyFolderFiles[0].Name = "06052022_Something_Academy_20052022";
+            academyFolderFiles[0].CreatedTime = new DateTime(2022, 2, 20);
+            academyFolderFiles[1].Name = "11022023_Something_Academy_25022023";
+            academyFolderFiles[1].CreatedTime = new DateTime(2023, 2, 25);
 
-            var newNameFile1 = "HousingBenefitFile20220513.dat";
-            var newNameFile2 = "HousingBenefitFile20230218.dat";
+            var newNameFile1 = "HousingBenefitFile20220221.dat";
+            var newNameFile2 = "HousingBenefitFile20230227.dat";
 
             var nameChangeRegister = new Dictionary<string, string>() {
-                { oldNameFile1, newNameFile1 },
-                { oldNameFile2, newNameFile2 }
+                { academyFolderFiles[0].Name, newNameFile1 },
+                { academyFolderFiles[1].Name, newNameFile2 }
```

Comment:
Does the tested code cater for more than 1 file in the Academy source folder?
In other words if there are two files dropped in the folder with the same derived "following Monday" date, will they both be copied to the HFS HB folder without a naming conflict?

Please confirm with a test.

Reply:
What functionality would we want for the case where more than 1 file is in the academy source folder?
I'll write a test to see what it currently does in the meantime, as that was difficult to do by inspection.

Reply:
I added a test to verify that it will call the file copy file method twice for two files in the same week. I believe that the real-life implication of that on Google Drive is not something we can test in this code-base, we'd have to test that in development or staging or with an end-to-end test because it's based on Google's implementation.